lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEFYarn.cs

```diff
@@ -115,6 +121,39 @@ private void LogApplicationReport()
         }
 
         /// <summary>
+        /// Get application report and log for given appId
+        /// </summary>
+        private void LogApplicationReport(string appId)
+        {
+            Logger.Log(Level.Info, "Getting Application report...");
+            var apps = _reefClient.GetApplicationReports().Result;
+            Logger.Log(Level.Info, "Got Application report: {0}", apps.Count);
+            IApplicationReport report;
+            apps.TryGetValue(appId, out report);
+            if (report != null)
+            {
+                Logger.Log(Level.Info, "Application report -- AppId {0}: {1}.", appId, report.ToString());
+            }
+        }
+
+        /// <summary>
+        /// Kill Job Application.
+        /// </summary>
+        /// <param name="appId">Application id to kill</param>
+        private void KillJobApplication(string appId)
+        {
+            try
+            {
+                var state = _reefClient.KillJobApplication(appId);
+                Logger.Log(Level.Info, "Application killed with state: ", state);
+            }
+            catch (Exception e)
+            {
+                Logger.Log(Level.Error, "Fail to kill Application with exception:", e);
```

Comment:
The catch can be removed. It is up to the client how they want to handle the exception.